demonstrations/tutorial_qnn_module_torch.py

```diff
@@ -86,7 +86,10 @@
 @qml.qnode(dev)
 def qnode(inputs, weights):
     qml.templates.AngleEmbedding(inputs, wires=range(n_qubits))
-    qml.templates.BasicEntanglerLayers(weights, wires=range(n_qubits))
+    for weights_layer in weights:
```

Comment:
```suggestion
    # explicit `qml.templates.BasicEngtnglerLayers` until a bug is fixed
    for weights_layer in weights:
```

Maybe just a comment here that there is usually a better way of doing this.

Reply:
:thinking: I see your point, although wonder if it may confuse the reader - unless they also look at the `Keras` tutorial they may not be expecting to see `BasicEntanglerLayers`.